packages/opera.rb

```diff
@@ -17,10 +17,10 @@ class Opera < Package
 
 
   def self.build
-    system "wget https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera-stable_#{version}_amd64.deb") ) == '08ec3d1f1b142db4acaea9961687ad95db123f8567c07b0157e350aeed26a56d'
-    system "alien -tc opera-stable_#{version}_amd64.deb"
-    system "tar xvf opera-stable-#{version}.tgz"
+    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb --output opera.deb"
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera.deb") ) == `curl -L https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`
+    system "alien -tc opera.deb > /dev/null"
+    system "tar xvf opera-stable-#{version}.tgz > /dev/null"
```

Comment:
I would rather not use `> /dev/null`.  This will hide any errors if they occur.

Reply:
Line 24 has `> /dev/null`.

Reply:
Only `tar` output hide now, as `tar` won't occur error for the most time

Reply:
`> /dev/null` won't hide error (it will hide `stdout` but keep `stderr`)

Reply:
I get that but if there is an error, don't you need `stdout` to see where the error occurred?